package.json

```diff
@@ -159,19 +160,16 @@
     "gzip-size": "^3.0.0",
     "http-server": "^0.9.0",
     "istanbul": "0.4.2",
-    "jasmine": "2.4.1",
-    "jasmine-ajax": "3.2.0",
-    "jasmine-core": "2.4.1",
     "karma": "^0.13.22",
     "karma-browserify": "5.0.2",
     "karma-chrome-launcher": "0.2.2",
-    "karma-jasmine": "0.3.7",
```

Comment:
We don't have to replace this with [`karma-mocha`](https://github.com/karma-runner/karma-mocha)?